tests/test_auto_annotator.py

```diff
@@ -9,14 +9,47 @@
 """
 import pytest
 
-from mindmeld.auto_annotator import SpacyAnnotator
+from mindmeld.auto_annotator import MultiLingualAnnotator
 
 
 @pytest.fixture(scope="module")
-def spacy_annotator(kwik_e_mart_app_path):
-    return SpacyAnnotator(kwik_e_mart_app_path)
+def en_mla(kwik_e_mart_app_path):
+    return MultiLingualAnnotator(
+        app_path=kwik_e_mart_app_path, language="en", translator="NoOpTranslator"
+    )
 
 
+@pytest.fixture(scope="module")
```

Comment:
Is there a way to test `translator="GoogleTranslator"`without calling google translate? maybe through a mock?

Reply:
Added a TODO statement in test_auto_annotator.py as discussed